apps/website/src/components/HomepageBanner.astro

```diff
@@ -0,0 +1,65 @@
+---
+import RightArrowIcon from '~/icons/RightArrowIcon.astro';
+
+type Props = {
+  href: string;
+  [key: string]: unknown;
+};
+
+const { href, ...props } = Astro.props;
+---
+
+<aside {...props}>
+  <a href={href}>
+    <slot />
+  </a>
+
+  <RightArrowIcon class='arrow' lineProps={{ class: 'line' }} />
+</aside>
+
+<style lang='scss'>
```

Comment:
There seems to be no focus outline when focusing on the banner link

Reply:
yikes! good catch, fixed.
![image](https://github.com/iTwin/iTwinUI/assets/9084735/8141b66e-389d-4dc7-ac53-1e6bc9fd05d7)

note to self: `all: unset` is a terrible hammer